### PR TITLE
doc: add minutes 31 May 2023

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

**Review by RafaelGSS — APPROVED** [review]

File: meetings/2023-05-31.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+* James Snell @jasnell (voting member)
+* Chengzhong Wu @legendecas (voting member)
+* Michael Dawson @mhdawson (voting member)
+* Moshe Atlow @MoLow (voting member)
+* Rafael Gonzaga @RafaelGSS (voting member)
+* Darshan Sen @RaisinTen (voting member)
+* Richard Lau @richardlau (voting member)
+* Rich Trott @Trott (voting member)
+
+
```
Comment (Trott) [inline meetings/2023-05-31.md:22]:
```suggestion
```

File: meetings/2023-05-31.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+* Antoine du Hamel @aduh95 (voting member)
+* Ruben Bridgewater @BridgeAR (voting member)
+* Gireesh Punathil @gireeshpunathil (voting member)
+* James Snell @jasnell (voting member)
+* Chengzhong Wu @legendecas (voting member)
+* Michael Dawson @mhdawson (voting member)
+* Moshe Atlow @MoLow (voting member)
+* Rafael Gonzaga @RafaelGSS (voting member)
+* Darshan Sen @RaisinTen (voting member)
+* Richard Lau @richardlau (voting member)
```
Comment (Trott) [inline meetings/2023-05-31.md:19]: Although arrived very late, @tniessen was at the meeting.

```suggestion
* Richard Lau @richardlau (voting member)
* Tobias Nießen @tniessen (voting member)
```

**Review by MoLow — APPROVED** [review]

**Review by gireeshpunathil — APPROVED** [review]

File: meetings/2023-05-31.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+## Strategic Initiatives
+
+* Antoine - Not update this week.
+* Quic/HTTPs - James, making progress more PRs have landed, will be a number more incremental ones, will be getting smaller in size.
+* Next-10
+  * hoping to get, data for recent survey into Next-10 repo in next week
+  * Have reached out to some newsletter in context of - <https://github.com/nodejs/node/blob/main/doc/contributing/sharing-project-news.md>, will be good if we start adding news more regularly now.
+  * Working on serverless deep dive session planned for June 22nd. If you know people who would be interested please at mention in <https://github.com/nodejs/next-10/issues/200> or let us know so we can contact them
+
+* SEA - Darshan - open PR to cash byte code for SEA - <https://github.com/nodejs/node/pull/48191>, updates tooling to bungle in byte code instead of JavaScript.
```
Comment (RaisinTen) [inline meetings/2023-05-31.md:60]:
```suggestion
* SEA - Darshan - open PR to cache byte code for SEA - <https://github.com/nodejs/node/pull/48191>, updates tooling to bundle in byte code along with JavaScript.
```

**Review by RaisinTen — APPROVED** [review]

File: meetings/2023-05-31.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+## Strategic Initiatives
+
+* Antoine - Not update this week.
+* Quic/HTTPs - James, making progress more PRs have landed, will be a number more incremental ones, will be getting smaller in size.
+* Next-10
+  * hoping to get, data for recent survey into Next-10 repo in next week
+  * Have reached out to some newsletter in context of - <https://github.com/nodejs/node/blob/main/doc/contributing/sharing-project-news.md>, will be good if we start adding news more regularly now.
+  * Working on serverless deep dive session planned for June 22nd. If you know people who would be interested please at mention in <https://github.com/nodejs/next-10/issues/200> or let us know so we can contact them
+
+* SEA - Darshan - open PR to cache byte code for SEA - <https://github.com/nodejs/node/pull/48191>, updates tooling to bundle in byte code along with JavaScript.
```
Comment (RaisinTen) [inline meetings/2023-05-31.md:60]:
```suggestion
* SEA - Darshan - open PR to cache byte code for SEA - <https://github.com/nodejs/node/pull/48191>, updates tooling to bundle in byte code along with JavaScript to improve the startup performance of the SEA.
```

**Pull request merged**